releasenotes/notes/40561.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+
+releaseNotes:
+  - |
+    **Fixed** disable `PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING` temporarily to fix some memory/CPU cost issues.
```

Comment:
we should not this breaks `istioctl x wait`. I would make this `**Removed** default `PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING` due to performance issues; this also makes `istioctl x wait` not functional`.